Sources/StreamVideoSwiftUI/CallView/LocalParticipantViewModifier.swift

```diff
@@ -10,16 +10,18 @@ import StreamVideo
 public struct LocalParticipantViewModifier: ViewModifier {
 
     private let localParticipant: CallParticipant
-    @StateObject private var microphoneChecker: MicrophoneChecker
+    @Injected(\.microphoneChecker) var microphoneChecker
```

Comment:
I think if you do:
```swift
@ObservedObject var microphoneChecker = InjectedValues[\.microphoneChecker]
```
It would simplify the code a bit. This is also ok, but would be nice to avoid storing extra state.